sections/recommendations.tex

```diff
@@ -33,11 +33,44 @@ \subsection*{Magic Arrows}
 Except for the Magic Arrow, each Spell appears exactly twice in the Spell Deck.
 After players prepare their starting M\&M Decks, it is recommended to remove Magic Arrows from the Spell Deck until no more than two remain.
 
+\subsection*{Guaranteed Settlement}
+
+When distributing Far Tiles (II--III) between players, each player's pool of Map Tiles must include one Settlement.
+
+\includegraphics[width=\linewidth]{\art/rogue.jpg}
+
+\includegraphics[width=\linewidth]{\art/medusa.jpg}
+
+\subsection*{War Machines as Artifacts}
+
+Shuffle one of each War Machine into the Artifact Deck before the start of the game.
+If using the optional Tournament rule of splitting the Artifact Deck, shuffle the War Machines into the Major Artifact Deck.
+
+\subsection*{Round One Mulligan}
+
+At the start of the \nth{1} Round, each player may reshuffle their hand of Cards back into their Deck of Might and Magic and draw a new starting hand of Cards once.
+
+\subsection*{Tier V--VII Combat on Hard+ Difficulty}
```

Comment:
That '+' sign looks weird on the renders.

Reply:
Fixed with math mode.